public/docs/ts/latest/guide/index.jade

```diff
@@ -22,7 +21,7 @@ table(width="100%")
   col(width="15%")
   col
   tr(style=top)
-    td <b>QuickStart</b>
+    td <b><a href="../quickstart">QuickStart</a></b>
```

Comment:
No, I don't think a self-link is helpful. We could maybe briefly mention in the text that that's where they are.